packages/gradbench/src/App.tsx

```diff
@@ -1,10 +1,151 @@
+      </p> */}
+
+      <div className="row">
+        <h2 className="subheading">Benchmarking</h2>
+        <div className="fillspace"></div>
+      </div>
+      <p className="text">
+        GradBench is designed to work across languages and domains. Functions
+        are written in <b>Adroit</b> and automatically translated a to tools
+        native language. The architexture allows for the easy addition of new
+        functions and tools.
       </p>
+
+      <div className="row">
+        <h2 className="subheading">Adroit</h2>
+        <div className="fillspace"></div>
+      </div>
+
+      <p className="text">Explain adroit</p>
+
+      <div className="row">
+        <h2 className="subheading">Currently Implemented</h2>
+        <div className="fillspace"></div>
+      </div>
+
+      <div className="container">
+        <table>
+          <tr>
+            <th className="no-border"></th>
+            <th className="tool">Autograd</th>
+            <th className="tool">DiffSharp</th>
+            <th className="tool">Jax</th>
+            <th className="tool">MyGrad</th>
+            <th className="tool">PyTorch</th>
+            <th className="tool">SciLean</th>
+            <th className="tool">Tapenade</th>
+            <th className="tool">TensorFlow</th>
+            <th className="tool">Zygote</th>
+          </tr>
+          <tr>
+            <td className="module">Hello</td>
+            <td className="emoji">&#10004;</td>
+            <td className="emoji">&#10004;</td>
+            <td className="emoji">&#10004;</td>
+            <td className="emoji">&#10004;</td>
+            <td className="emoji">&#10004;</td>
+            <td className="emoji">&#10004;</td>
+            <td className="emoji">&#10004;</td>
+            <td className="emoji">&#10004;</td>
+            <td className="emoji">&#10004;</td>
+          </tr>
+          <tr>
+            <td className="module">GMM</td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td className="emoji">&#10004;</td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+          </tr>
+          <tr>
+            <td className="module">BA</td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td className="emoji">&#8987;</td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+          </tr>
+          <tr>
+            <td className="module">HT</td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+          </tr>
+        </table>
+        <div>
+          <p className="text">Table explanation</p>
+        </div>
+      </div>
+
+      <div className="row">
+        <h2 className="subheading">Links</h2>
```

Comment:
Instead of having a separate "links" section, could we have each of the individual row headers in the table above be links?